src/Analysis/PackageDef.hs

```diff
@@ -50,13 +50,13 @@ instance (PackageDefStrategy syntax ~ strategy, HasPackageDefWithStrategy strate
 -- | Types for which we can produce a customized 'PackageDef'. This returns in 'Maybe' so that some values can be opted out (e.g. anonymous functions).
 class CustomHasPackageDef syntax where
   -- | Produce a customized 'PackageDef' for a given syntax node.
-  customToPackageDef :: (Foldable whole) => Blob -> Location -> syntax (Term whole Location, Maybe PackageDef) -> Maybe PackageDef
+  customToPackageDef :: (Foldable whole) => Blob -> Loc -> syntax (Term whole Loc, Maybe PackageDef) -> Maybe PackageDef
 
 
 instance CustomHasPackageDef Language.Go.Syntax.Package where
   customToPackageDef Blob{..} _ (Language.Go.Syntax.Package (Term (In fromAnn _), _) _)
     = Just $ PackageDef (getSource fromAnn)
-    where getSource = toText . flip Source.slice blobSource . locationByteRange
+    where getSource = toText . Source.slice blobSource . byteRange
```

Comment:
We flipped `Source.slice`’s definition since we so frequently had to flip its uses instead.